practical/swarm_demo/install.sh

```diff
@@ -5,19 +5,28 @@
 #
 # USAGE: chmod +x ./install.sh && ./install.sh
 #
-# DESCRIPTION: Tears down an existing Swarm hosts and
-#              provisions a fresh cluster.
-
+# DESCRIPTION: Creates a Docker Swarm cluster with a discovery service node (Consul),
+#              a master and two slave nodes and runs the CA674 demo app on top of it.
+#              The CA674 demo is comprised of a back-end store (redis) and front-end
+#              www implementation (python) scaled to initially two instances.
+#              A load balancer is used to randomize access to each front-end container
+#              and unify the entry point of the app.
+#
 #===================================================================================
 
-VM=local
+function teardown_swarm_machines {
+  echo "Removing old Swarm machines..."
+  docker-machine stop keystore
```

Comment:
None of these commands will work if any of these machines aren't already created. Also, you don't need to stop the machines first. 

```
docker-machine status keystore && docker-machine rm -f keystore # if the first commands returns successfully the rm the key store machine
...
```
docker-machine stop swarm-master # if this machine doesn't exist then the `teardown_swarm_machines` will return straight away and the rest of the bellow commands don't get terminated
...
```
docker-machine status keystore && docker-machine rm -f keystore # if the first commands returns successfully the rm the key store machine
```